src/App.css

```diff
@@ -1,28 +1,43 @@
+@import url("https://fonts.googleapis.com/css?family=Nunito");
+body {
+  font-family: 'Nunito', sans-serif; }
+
 .App {
-  text-align: center;
-}
+  text-align: center; }
```

Comment:
Please follow [Google CSS Guide](https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules). The closing bracket should be in the next line.

If this is a generated file from App.scss file, could you please minify it.

Reply:
@PrabhanshuAttri This is being automatically set by the css watcher file from scss file actually when running npm start.

Reply:
@PrabhanshuAttri For this minification part, we can open another issue, that might help.

Reply:
I've fixed this. Please take a pull from the upstream for the latest code.

Reply:
@divyamrastogi sure.